File: Csharp/URI/01-Iniciantes/04-Nivel/1002.cs
using System;
namespace Exe1002.URI
{
    public class Exe1002
    {
        public  Exe1002()
        {
            const double pi = 3.14159;
            double ray, area;
            Console.WriteLine("Digite o tamanho do raio");
            ray = double.Parse(Console.ReadLine());
            area = pi * Math.Pow(ray , 2);
            Console.WriteLine("A={0:0.0000} ", area);
        }
    }
}

File: Cplusplus/URI/01-iniciantes/04-Nivel/1002.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
 
using namespace std;
#define pi 3.14159
 
int main()
{
    double ray,area;    
    cout << fixed;   
    cout << "Digite o tamanho do raio: ";
    cin >>  ray >> setprecision(2);
    area = pi * pow(ray , 2);
    cout << "A=" << setprecision(4) << area << endl; 
}